gqlerrors/decode.go

```diff
@@ -0,0 +1,38 @@
+package gqlerrors
+
+import (
+	"reflect"
+
+	"github.com/gofrs/uuid"
+	"github.com/mitchellh/mapstructure"
+)
+
+// Decode an objects
+func Decode(input interface{}, output interface{}) error {
```

Comment:
We probably don't want to expose this kind of functions which are used internally.